cli/src/pcluster/config/cluster_config.py

```diff
@@ -1341,6 +1342,10 @@ def _register_validators(self):
         )
         # TODO add InstanceTypesBaseAMICompatibleValidator
 
+        for storage in self.shared_storage:
+            if isinstance(storage, SharedFsx):
+                self._register_validator(FsxSchedulerValidator, scheduler=self.scheduling.scheduler)
```

Comment:
It might be cleanner to register the validator in `_register_storage_validators`